### PR TITLE
Update install script huge page setup

### LINKED ISSUE #83: install.sh doesn't remove the old igb_uio module and install the new one when compiling dpdk
## Bug Report

**Current Behavior**
`install.sh` doesn't reinstall igb_uio module generated by dpdk if it finds the system has already installed one. Since dpdk provides the driver of igb_uio which might be often updated, the old igb_uio module should be removed and replaced by the dpdk's. This can cause a run-time error for running `dpdk_iface_main` to create a dpdk-iface device.

**Expected behavior/code**
igb_uio module should be removed and re-installed at each time of compiling dpdk

**Steps to reproduce**

- Apply for a new machine or reboot a machine from cloudlab, the system has already installed a default igb_uio, use `grep -m 1 "igb_uio" /proc/modules` to display:        

``` igb_uio 2446317 0 - Live 0x0000000000000000 (OX) ```

- Then run `install.sh` to compile dpdk, and use `grep -m 1 "igb_uio" /proc/modules` to display again:     

``` igb_uio 2446317 0 - Live 0x0000000000000000 (OX) ```

The old igb_uio module was not replaced by the new one.

-  Run ` setup_mtcp_onvm_env.sh ` to create a dpdk-iface device, an error happened:
```
xiaosuGW@node2:~/mtcp$ ./setup_mtcp_onvm_env.sh
Checking ldflags.txt...
RTE_SDK env variable is set to /users/xiaosuGW/openNetVM-dev/dpdk
RTE_TARGET env variable is set to x86_64-native-linuxapp-gcc
Are you using an Intel NIC (y/n)? y
Creating dpdk interface entries
make -C /lib/modules/3.13.0-117-generic/build/ M=/users/xiaosuGW/mtcp/dpdk-iface-kmod modules
make[1]: Entering directory `/usr/src/linux-headers-3.13.0-117-generic'
  Building modules, stage 2.
  MODPOST 1 modules
make[1]: Leaving directory `/usr/src/linux-headers-3.13.0-117-generic'
make -C /lib/modules/3.13.0-117-generic/build/ M=/users/xiaosuGW/mtcp/dpdk-iface-kmod modules
make[1]: Entering directory `/usr/src/linux-headers-3.13.0-117-generic'
  Building modules, stage 2.
  MODPOST 1 modules
make[1]: Leaving directory `/usr/src/linux-headers-3.13.0-117-generic'
sudo ./dpdk_iface_main
Removing existing device node entry... not present.
Creating device node entry... done.
Setting permissions on the device node entry... done.
Scanning the system for dpdk-compatible devices... done.
Clearing previous entries
ioctl call failed!
make: *** [run] Error 1

```

**Environment**

- OS:  cloudlab machines
- onvm version:  develop branch with commit `e305489b76c868e722470c7799625de90d5f37bb`
- dpdk version:   18.11.0
- mtcp:  branch `onvm_build_readme_fix` 

**Possible Solution**
Removing the old igb_uio and re-installing the new one under `$RTE_SDK/$RTE_TARGET/kmod` can resolve this problem.

### PR DESCRIPTION
**Pull request by koolzz**
Uses the same functions that dpdk uses to set up huge pages, with minor adjustments to onvm.

## Summary:
- Adds better huge page setup to the `install.sh` and `setup_environment.sh` scripts.
   - Properly mounts hp, which makes our `setup_environment.sh` equivalent to using the `dpdk-setup.sh` script.
   - Sets memory for all NUMA nodes, instead of just `node0` (this could actually resolve some hugepage bugs)
- Removes the old igb_uio module on `install.sh`

**Usage:**
Install onvm, use the setup_script.

| This PR includes         |          |
| ------------------------ | -------- |
| Resolves issues          |  Fixes #83
| Breaking API changes     |
| Internal API changes     |
| Usability improvements   |  :100: 
| Bug fixes                | :+1: 
| New functionality        |
| New NF/onvm_mgr args     | 
| Changes to starting NFs  |  
| Dependency updates       | 
| Web stats updates        | 

## Merging notes:
 - Dependencies: None  

**TODO before merging :**
 - [x] PR is ready for review

## Test Plan:
Run both, be sure it doesn't break. Check if it actually resolves #83. 

## Review: 
Assigned to @rskennedy

### CODE REVIEW

**Comment by onvm** [comment]
> In response to PR creation

## CI Message

Your results will arrive shortly

**Comment by onvm** [comment]
> In response to PR creation

## CI Message

Run successful see results:
[Results from nimbnode30]
Median TX pps for Speed Tester: 35260194

## Linter Failed

examples/aes_decrypt/aes.c:523:  If an else has a brace on one side, it should have it on both  [readability/braces] [5]
examples/aes_decrypt/aes.c:1024:  If an else has a brace on one side, it should have it on both  [readability/braces] [5]
examples/aes_decrypt/aes.c:1028:  If an else has a brace on one side, it should have it on both  [readability/braces] [5]
examples/aes_decrypt/aes.c:1081:  If an else has a brace on one side, it should have it on both  [readability/braces] [5]
examples/aes_decrypt/aes.c:1087:  If an else has a brace on one side, it should have it on both  [readability/braces] [5]
Total errors found: 5
examples/aes_encrypt/aes.c:523:  If an else has a brace on one side, it should have it on both  [readability/braces] [5]
examples/aes_encrypt/aes.c:1024:  If an else has a brace on one side, it should have it on both  [readability/braces] [5]
examples/aes_encrypt/aes.c:1028:  If an else has a brace on one side, it should have it on both  [readability/braces] [5]
examples/aes_encrypt/aes.c:1081:  If an else has a brace on one side, it should have it on both  [readability/braces] [5]
examples/aes_encrypt/aes.c:1087:  If an else has a brace on one side, it should have it on both  [readability/braces] [5]
Total errors found: 5
examples/arp_response/arp_response.c:144:  Missing space before ( in switch(  [whitespace/parens] [5]
Total errors found: 1
examples/bridge/bridge.c:135:  Missing space before ( in if(  [whitespace/parens] [5]
examples/bridge/bridge.c:138:  If an else has a brace on one side, it should have it on both  [readability/braces] [5]
examples/bridge/bridge.c:156:  If an else has a brace on one side, it should have it on both  [readability/braces] [5]
Total errors found: 3
examples/flow_table/flow_table.c:103:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/flow_table.c:198:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/flow_table.c:216:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/flow_table.c:224:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/flow_table.c:231:  If an else has a brace on one side, it should have it on both  [readability/braces] [5]
examples/flow_table/flow_table.c:238:  If an else has a brace on one side, it should have it on both  [readability/braces] [5]
Total errors found: 6
examples/flow_table/msgbuf.c:71:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/msgbuf.c:73:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/msgbuf.c:83:  Missing space before ( in while(  [whitespace/parens] [5]
examples/flow_table/msgbuf.c:86:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/msgbuf.c:89:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/msgbuf.c:106:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/msgbuf.c:108:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/msgbuf.c:116:  Missing space before ( in while(  [whitespace/parens] [5]
examples/flow_table/msgbuf.c:119:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/msgbuf.c:125:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/msgbuf.c:138:  Extra space before last semicolon. If this should be an empty statement, use {} instead.  [whitespace/semicolon] [5]
examples/flow_table/msgbuf.c:140:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/msgbuf.c:149:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/msgbuf.c:157:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/msgbuf.c:159:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/msgbuf.c:162:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/msgbuf.c:169:  Missing space before ( in while(  [whitespace/parens] [5]
Total errors found: 17
examples/flow_table/sdn.c:99:  Extra space before last semicolon. If this should be an empty statement, use {} instead.  [whitespace/semicolon] [5]
examples/flow_table/sdn.c:125:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/sdn.c:127:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/sdn.c:147:  If an else has a brace on one side, it should have it on both  [readability/braces] [5]
examples/flow_table/sdn.c:147:  Missing space before {  [whitespace/braces] [5]
examples/flow_table/sdn.c:153:  Missing space before ( in while(  [whitespace/parens] [5]
examples/flow_table/sdn.c:160:  Missing space before {  [whitespace/braces] [5]
examples/flow_table/sdn.c:235:  If an else has a brace on one side, it should have it on both  [readability/braces] [5]
examples/flow_table/sdn.c:239:  If an else has a brace on one side, it should have it on both  [readability/braces] [5]
examples/flow_table/sdn.c:279:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/sdn.c:418:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/sdn.c:423:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/sdn.c:493:  If an else has a brace on one side, it should have it on both  [readability/braces] [5]
examples/flow_table/sdn.c:501:  If an else has a brace on one side, it should have it on both  [readability/braces] [5]
examples/flow_table/sdn.c:538:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/sdn.c:542:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/sdn.c:545:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/sdn.c:547:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/sdn.c:555:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/sdn.c:569:  Missing space before ( in while(  [whitespace/parens] [5]
Total errors found: 20
examples/flow_table/setupconn.c:82:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/setupconn.c:89:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/setupconn.c:94:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/setupconn.c:102:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/setupconn.c:110:  Missing space before {  [whitespace/braces] [5]
examples/flow_table/setupconn.c:112:  Missing space before {  [whitespace/braces] [5]
examples/flow_table/setupconn.c:122:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/setupconn.c:140:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/setupconn.c:149:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/setupconn.c:156:  Missing space before ( in if(  [whitespace/parens] [5]
Total errors found: 10
examples/flow_tracker/flow_tracker.c:108:  Missing space before ( in switch(  [whitespace/parens] [5]
examples/flow_tracker/flow_tracker.c:210:  If an else has a brace on one side, it should have it on both  [readability/braces] [5]
Total errors found: 2
examples/scaling_example/scaling.c:312:  Missing space before ( in while(  [whitespace/parens] [5]
Total errors found: 1
onvm/onvm_mgr/main.c:98:  Mismatching spaces inside () in while  [whitespace/parens] [5]
Total errors found: 1
onvm/onvm_mgr/onvm_args.c:166:  Missing space before ( in if(  [whitespace/parens] [5]
onvm/onvm_mgr/onvm_args.c:166:  Missing space before {  [whitespace/braces] [5]
onvm/onvm_mgr/onvm_args.c:172:  Missing space before ( in if(  [whitespace/parens] [5]
onvm/onvm_mgr/onvm_args.c:172:  Missing space before {  [whitespace/braces] [5]
onvm/onvm_mgr/onvm_args.c:328:  Missing space before ( in if(  [whitespace/parens] [5]
onvm/onvm_mgr/onvm_args.c:352:  Missing space before {  [whitespace/braces] [5]
onvm/onvm_mgr/onvm_args.c:357:  Missing space before ( in if(  [whitespace/parens] [5]
Total errors found: 7
onvm/onvm_mgr/onvm_init.c:222:  Never use sprintf. Use snprintf instead.  [runtime/printf] [5]
Total errors found: 1
onvm/onvm_mgr/onvm_nf.c:182:  Missing space before ( in if(  [whitespace/parens] [5]
onvm/onvm_mgr/onvm_nf.c:255:  Missing space before ( in if(  [whitespace/parens] [5]
Total errors found: 2
onvm/onvm_mgr/onvm_stats.c:183:  Missing space before ( in if(  [whitespace/parens] [5]
onvm/onvm_mgr/onvm_stats.c:183:  Missing space before {  [whitespace/braces] [5]
Total errors found: 2
onvm/onvm_nflib/onvm_config_common.c:52:  Missing space before {  [whitespace/braces] [5]
onvm/onvm_nflib/onvm_config_common.c:112:  Missing space before ( in for(  [whitespace/parens] [5]
onvm/onvm_nflib/onvm_config_common.c:112:  Missing space before {  [whitespace/braces] [5]
onvm/onvm_nflib/onvm_config_common.c:417:  Never use sprintf. Use snprintf instead.  [runtime/printf] [5]
onvm/onvm_nflib/onvm_config_common.c:433:  Never use sprintf. Use snprintf instead.  [runtime/printf] [5]
onvm/onvm_nflib/onvm_config_common.c:487:  Never use sprintf. Use snprintf instead.  [runtime/printf] [5]
Total errors found: 6
onvm/onvm_nflib/onvm_flow_dir.c:64:  Missing space before ( in if(  [whitespace/parens] [5]
onvm/onvm_nflib/onvm_flow_dir.c:95:  Missing space before {  [whitespace/braces] [5]
onvm/onvm_nflib/onvm_flow_dir.c:103:  Missing space before {  [whitespace/braces] [5]
onvm/onvm_nflib/onvm_flow_dir.c:111:  Missing space before {  [whitespace/braces] [5]
onvm/onvm_nflib/onvm_flow_dir.c:128:  Missing space before {  [whitespace/braces] [5]
onvm/onvm_nflib/onvm_flow_dir.c:143:  Missing space before {  [whitespace/braces] [5]
onvm/onvm_nflib/onvm_flow_dir.c:151:  Missing space before {  [whitespace/braces] [5]
onvm/onvm_nflib/onvm_flow_dir.c:159:  Missing space before {  [whitespace/braces] [5]
onvm/onvm_nflib/onvm_flow_dir.c:176:  Missing space before {  [whitespace/braces] [5]
Total errors found: 9
onvm/onvm_nflib/onvm_nflib.c:341:  Missing space before ( in if(  [whitespace/parens] [5]
onvm/onvm_nflib/onvm_nflib.c:344:  Missing space before ( in if(  [whitespace/parens] [5]
onvm/onvm_nflib/onvm_nflib.c:347:  Missing space before ( in if(  [whitespace/parens] [5]
onvm/onvm_nflib/onvm_nflib.c:351:  Missing space before ( in if(  [whitespace/parens] [5]
onvm/onvm_nflib/onvm_nflib.c:354:  Missing space before ( in if(  [whitespace/parens] [5]
onvm/onvm_nflib/onvm_nflib.c:357:  Missing space before ( in if(  [whitespace/parens] [5]
onvm/onvm_nflib/onvm_nflib.c:584:  Missing space before ( in switch(  [whitespace/parens] [5]
onvm/onvm_nflib/onvm_nflib.c:743:  Missing space before ( in if(  [whitespace/parens] [5]
onvm/onvm_nflib/onvm_nflib.c:754:  Missing space before ( in if(  [whitespace/parens] [5]
onvm/onvm_nflib/onvm_nflib.c:821:  Missing space before {  [whitespace/braces] [5]
Total errors found: 10
onvm/onvm_nflib/onvm_pkt_helper.c:313:  Missing space before ( in switch(  [whitespace/parens] [5]
onvm/onvm_nflib/onvm_pkt_helper.c:376:  Missing space before {  [whitespace/braces] [5]
Total errors found: 2

**Comment by rskennedy** [comment]
@rskennedy Going to test this in addition to @dennisafa

File: scripts/dpdk_helper_scripts.sh [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    fi
+}
+
+#
+# Removes hugepage filesystem.
+#
+remove_mnt_huge()
+{
+    echo "Unmounting /mnt/huge and removing directory"
+    grep -s '/mnt/huge' /proc/mountsi > /dev/null
```
Comment (rskennedy) [inline scripts/dpdk_helper_scripts.sh]: Is this a typo? not sure that mountsi is a file

Comment (koolzz) [reply scripts/dpdk_helper_scripts.sh]: yeah thats definitely not a file, good catch it should just be
`grep -s '/mnt/huge' /proc/mounts > /dev/null`

Comment (koolzz) [reply scripts/dpdk_helper_scripts.sh]: @rskennedy resolved

**Comment by koolzz** [comment]
@onvm encryption test

**Comment by koolzz** [comment]
@onvm again

**Comment by koolzz** [comment]
@onvm good to go?

**Comment by onvm** [comment]
> @onvm good to go?

## CI Message

Your results will arrive shortly

**Comment by onvm** [comment]
> @onvm good to go?

## CI Message

Run successful see results:
[Results from nimbnode30]
Median TX pps for Speed Tester: 35236201

## Linter Failed

examples/aes_decrypt/aes.c:523:  If an else has a brace on one side, it should have it on both  [readability/braces] [5]
examples/aes_decrypt/aes.c:1024:  If an else has a brace on one side, it should have it on both  [readability/braces] [5]
examples/aes_decrypt/aes.c:1028:  If an else has a brace on one side, it should have it on both  [readability/braces] [5]
examples/aes_decrypt/aes.c:1081:  If an else has a brace on one side, it should have it on both  [readability/braces] [5]
examples/aes_decrypt/aes.c:1087:  If an else has a brace on one side, it should have it on both  [readability/braces] [5]
Total errors found: 5
examples/aes_encrypt/aes.c:523:  If an else has a brace on one side, it should have it on both  [readability/braces] [5]
examples/aes_encrypt/aes.c:1024:  If an else has a brace on one side, it should have it on both  [readability/braces] [5]
examples/aes_encrypt/aes.c:1028:  If an else has a brace on one side, it should have it on both  [readability/braces] [5]
examples/aes_encrypt/aes.c:1081:  If an else has a brace on one side, it should have it on both  [readability/braces] [5]
examples/aes_encrypt/aes.c:1087:  If an else has a brace on one side, it should have it on both  [readability/braces] [5]
Total errors found: 5
examples/arp_response/arp_response.c:144:  Missing space before ( in switch(  [whitespace/parens] [5]
Total errors found: 1
examples/bridge/bridge.c:135:  Missing space before ( in if(  [whitespace/parens] [5]
examples/bridge/bridge.c:138:  If an else has a brace on one side, it should have it on both  [readability/braces] [5]
examples/bridge/bridge.c:156:  If an else has a brace on one side, it should have it on both  [readability/braces] [5]
Total errors found: 3
examples/flow_table/flow_table.c:103:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/flow_table.c:198:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/flow_table.c:216:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/flow_table.c:224:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/flow_table.c:231:  If an else has a brace on one side, it should have it on both  [readability/braces] [5]
examples/flow_table/flow_table.c:238:  If an else has a brace on one side, it should have it on both  [readability/braces] [5]
Total errors found: 6
examples/flow_table/msgbuf.c:71:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/msgbuf.c:73:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/msgbuf.c:83:  Missing space before ( in while(  [whitespace/parens] [5]
examples/flow_table/msgbuf.c:86:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/msgbuf.c:89:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/msgbuf.c:106:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/msgbuf.c:108:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/msgbuf.c:116:  Missing space before ( in while(  [whitespace/parens] [5]
examples/flow_table/msgbuf.c:119:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/msgbuf.c:125:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/msgbuf.c:138:  Extra space before last semicolon. If this should be an empty statement, use {} instead.  [whitespace/semicolon] [5]
examples/flow_table/msgbuf.c:140:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/msgbuf.c:149:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/msgbuf.c:157:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/msgbuf.c:159:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/msgbuf.c:162:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/msgbuf.c:169:  Missing space before ( in while(  [whitespace/parens] [5]
Total errors found: 17
examples/flow_table/sdn.c:99:  Extra space before last semicolon. If this should be an empty statement, use {} instead.  [whitespace/semicolon] [5]
examples/flow_table/sdn.c:125:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/sdn.c:127:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/sdn.c:147:  If an else has a brace on one side, it should have it on both  [readability/braces] [5]
examples/flow_table/sdn.c:147:  Missing space before {  [whitespace/braces] [5]
examples/flow_table/sdn.c:153:  Missing space before ( in while(  [whitespace/parens] [5]
examples/flow_table/sdn.c:160:  Missing space before {  [whitespace/braces] [5]
examples/flow_table/sdn.c:235:  If an else has a brace on one side, it should have it on both  [readability/braces] [5]
examples/flow_table/sdn.c:239:  If an else has a brace on one side, it should have it on both  [readability/braces] [5]
examples/flow_table/sdn.c:279:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/sdn.c:418:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/sdn.c:423:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/sdn.c:493:  If an else has a brace on one side, it should have it on both  [readability/braces] [5]
examples/flow_table/sdn.c:501:  If an else has a brace on one side, it should have it on both  [readability/braces] [5]
examples/flow_table/sdn.c:538:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/sdn.c:542:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/sdn.c:545:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/sdn.c:547:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/sdn.c:555:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/sdn.c:569:  Missing space before ( in while(  [whitespace/parens] [5]
Total errors found: 20
examples/flow_table/setupconn.c:82:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/setupconn.c:89:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/setupconn.c:94:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/setupconn.c:102:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/setupconn.c:110:  Missing space before {  [whitespace/braces] [5]
examples/flow_table/setupconn.c:112:  Missing space before {  [whitespace/braces] [5]
examples/flow_table/setupconn.c:122:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/setupconn.c:140:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/setupconn.c:149:  Missing space before ( in if(  [whitespace/parens] [5]
examples/flow_table/setupconn.c:156:  Missing space before ( in if(  [whitespace/parens] [5]
Total errors found: 10
examples/flow_tracker/flow_tracker.c:108:  Missing space before ( in switch(  [whitespace/parens] [5]
examples/flow_tracker/flow_tracker.c:210:  If an else has a brace on one side, it should have it on both  [readability/braces] [5]
Total errors found: 2
examples/scaling_example/scaling.c:312:  Missing space before ( in while(  [whitespace/parens] [5]
Total errors found: 1
onvm/onvm_mgr/main.c:98:  Mismatching spaces inside () in while  [whitespace/parens] [5]
Total errors found: 1
onvm/onvm_mgr/onvm_args.c:166:  Missing space before ( in if(  [whitespace/parens] [5]
onvm/onvm_mgr/onvm_args.c:166:  Missing space before {  [whitespace/braces] [5]
onvm/onvm_mgr/onvm_args.c:172:  Missing space before ( in if(  [whitespace/parens] [5]
onvm/onvm_mgr/onvm_args.c:172:  Missing space before {  [whitespace/braces] [5]
onvm/onvm_mgr/onvm_args.c:328:  Missing space before ( in if(  [whitespace/parens] [5]
onvm/onvm_mgr/onvm_args.c:352:  Missing space before {  [whitespace/braces] [5]
onvm/onvm_mgr/onvm_args.c:357:  Missing space before ( in if(  [whitespace/parens] [5]
Total errors found: 7
onvm/onvm_mgr/onvm_init.c:222:  Never use sprintf. Use snprintf instead.  [runtime/printf] [5]
Total errors found: 1
onvm/onvm_mgr/onvm_nf.c:182:  Missing space before ( in if(  [whitespace/parens] [5]
onvm/onvm_mgr/onvm_nf.c:255:  Missing space before ( in if(  [whitespace/parens] [5]
Total errors found: 2
onvm/onvm_mgr/onvm_stats.c:183:  Missing space before ( in if(  [whitespace/parens] [5]
onvm/onvm_mgr/onvm_stats.c:183:  Missing space before {  [whitespace/braces] [5]
Total errors found: 2
onvm/onvm_nflib/onvm_config_common.c:52:  Missing space before {  [whitespace/braces] [5]
onvm/onvm_nflib/onvm_config_common.c:112:  Missing space before ( in for(  [whitespace/parens] [5]
onvm/onvm_nflib/onvm_config_common.c:112:  Missing space before {  [whitespace/braces] [5]
onvm/onvm_nflib/onvm_config_common.c:417:  Never use sprintf. Use snprintf instead.  [runtime/printf] [5]
onvm/onvm_nflib/onvm_config_common.c:433:  Never use sprintf. Use snprintf instead.  [runtime/printf] [5]
onvm/onvm_nflib/onvm_config_common.c:487:  Never use sprintf. Use snprintf instead.  [runtime/printf] [5]
Total errors found: 6
onvm/onvm_nflib/onvm_flow_dir.c:64:  Missing space before ( in if(  [whitespace/parens] [5]
onvm/onvm_nflib/onvm_flow_dir.c:95:  Missing space before {  [whitespace/braces] [5]
onvm/onvm_nflib/onvm_flow_dir.c:103:  Missing space before {  [whitespace/braces] [5]
onvm/onvm_nflib/onvm_flow_dir.c:111:  Missing space before {  [whitespace/braces] [5]
onvm/onvm_nflib/onvm_flow_dir.c:128:  Missing space before {  [whitespace/braces] [5]
onvm/onvm_nflib/onvm_flow_dir.c:143:  Missing space before {  [whitespace/braces] [5]
onvm/onvm_nflib/onvm_flow_dir.c:151:  Missing space before {  [whitespace/braces] [5]
onvm/onvm_nflib/onvm_flow_dir.c:159:  Missing space before {  [whitespace/braces] [5]
onvm/onvm_nflib/onvm_flow_dir.c:176:  Missing space before {  [whitespace/braces] [5]
Total errors found: 9
onvm/onvm_nflib/onvm_nflib.c:341:  Missing space before ( in if(  [whitespace/parens] [5]
onvm/onvm_nflib/onvm_nflib.c:344:  Missing space before ( in if(  [whitespace/parens] [5]
onvm/onvm_nflib/onvm_nflib.c:347:  Missing space before ( in if(  [whitespace/parens] [5]
onvm/onvm_nflib/onvm_nflib.c:351:  Missing space before ( in if(  [whitespace/parens] [5]
onvm/onvm_nflib/onvm_nflib.c:354:  Missing space before ( in if(  [whitespace/parens] [5]
onvm/onvm_nflib/onvm_nflib.c:357:  Missing space before ( in if(  [whitespace/parens] [5]
onvm/onvm_nflib/onvm_nflib.c:584:  Missing space before ( in switch(  [whitespace/parens] [5]
onvm/onvm_nflib/onvm_nflib.c:743:  Missing space before ( in if(  [whitespace/parens] [5]
onvm/onvm_nflib/onvm_nflib.c:754:  Missing space before ( in if(  [whitespace/parens] [5]
onvm/onvm_nflib/onvm_nflib.c:821:  Missing space before {  [whitespace/braces] [5]
Total errors found: 10
onvm/onvm_nflib/onvm_pkt_helper.c:313:  Missing space before ( in switch(  [whitespace/parens] [5]
onvm/onvm_nflib/onvm_pkt_helper.c:376:  Missing space before {  [whitespace/braces] [5]
Total errors found: 2

**Pull request merged**